zh/monitoring-by-tidb-monitor.md

```diff
@@ -0,0 +1,208 @@
+---
+title: 通过 TidbMonitor 监控 TiDB 集群
+category: how-to
+---
+
+# Kubernetes 上的 TiDB 集群监控
+
+在 Operator 1.1 版本及以上，我们可以通过简单的 CR 文件，即 TidbMonitor 来快速建立对 Kubernetes 集群上的 TiDB 集群的监控。
+
+
+## 快速上手
+
+> **注意：**
+>
+> 快速上手只是用来作为演示或是测试，请勿将以下配置部署在关键、生产环境。
+
+> 前置条件:
+
+    1. 已经安装了 Operator `v1.1.0-beta.1` 及以上版本，并且已经更新了相关版本的 CRD 文件
+    2. 已经设置了默认的 storageClass， 并保证其有足够的 PV (默认情况下需要 6 个PV)
+
+    这可以通过以下指令来验证:
+
+```shell
+$ kubectl get storageClass
+NAME                 PROVISIONER             RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+standard (default)   rancher.io/local-path   Delete          WaitForFirstConsumer   false                  14h
+```
```

Comment:
Please use two separate code blocks to show `copyable` commands and the returned information.
Also, we'd better add a sentence to tell users which value indicates the default storageClass is set.